components/Breadcrumbs.tsx

```diff
@@ -0,0 +1,32 @@
+import Container from 'react-bootstrap/Container'
+
+import { useTranslation } from 'next-i18next'
+
+export const Breadcrumbs: React.FC = () => {
+  const { t } = useTranslation('common')
+
+  return (
+    <Container className="breadcrumbs">
+      <a href="https://uio.edd.ca.gov">
+        <span className="text">{t('crumb-uio')}</span>
+      </a>
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="12"
+        height="12"
+        fill="#565C65"
+        className="bi bi-chevron-right"
+        viewBox="0 0 16 16"
+      >
+        <path
+          fillRule="evenodd"
+          d="M4.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L10.293 8 4.646 2.354a.5.5 0 0 1 0-.708z"
+        />
+      </svg>
+      <a href="https://uio.edd.ca.gov">
+        {/*  TODO what is the URL here? */}
```

Comment:
This could be worth a tiny ticket to make sure we don't ship the wrong thing.

Reply:
the component is marked as Rough Draft, so I think we'll be good!

Reply:
Yeah I think it'd be good to log this, Kat since you're figuring out content process with Nicole, can you create either a Github ticket or a spreadsheet with content things like this? UAT will want to compare every link to a reference doc (which, last year, we were the ones providing)

(This is something we can find ourselves by logging into BPO, but don't need to do that right now)

Reply:
Happy to be overruled! #142